test/allowlist/gcc/common.log

```diff
@@ -0,0 +1,5 @@
+# 
+# Timeout. Appears to be 32 bit?
```

Comment:
Can we make this rv32 only to test that?
`common.log -> rv32.log`

Reply:
updated